clients/drcov/postprocess/drcov2lcov.cpp

```diff
@@ -814,7 +814,8 @@ read_module_list(const char *buf, module_table_t ***tables, uint *num_mods)
             mod_table = (module_table_t *)MODULE_TABLE_IGNORE;
         else {
             if (op_pathmap.specified()) {
-                const char *tofind = op_pathmap.get_value().first.c_str();
+                const auto t = op_pathmap.get_value();
```

Comment:
s/auto/twostring_t/
(hard for reader to know what type this is: so IMHO "auto" makes the code harder to understand here)